docs/source/configuration/authorization.mdx

```diff
@@ -442,46 +441,38 @@ extend schema
     import: [..., "@policy"])
```

Comment:
I assume this will need to be fed 2.6 since 2.5 is out and doesn't support this yet